src/current/v23.1/architecture/replication-layer.md

```diff
@@ -220,7 +220,21 @@ Rebalancing is achieved by using a snapshot of a replica from the leaseholder, a
 
 #### Load-based replica rebalancing
 
-In addition to the rebalancing that occurs when nodes join or leave a cluster, replicas are also rebalanced automatically based on the relative load across the nodes within a cluster. For more information, see the `kv.allocator.load_based_rebalancing` and `kv.allocator.qps_rebalance_threshold` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}). Note that depending on the needs of your deployment, you can exercise additional control over the location of leases and replicas by [configuring replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}).
+In addition to the rebalancing that occurs when nodes join or leave a cluster, replicas are also rebalanced automatically across the cluster based on a combination of:
+
+1. Replica count
+2. CPU usage (if [`kv.allocator.load_based_rebalancing.objective`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-kv-allocator-load-based-rebalancing-objective) is set to `cpu`, which is the default in CockroachDB v23.1 and later)
+
+Load-based replica rebalancing operates in conjunction with [load-based splitting]({% link {{ page.version.version }}/load-based-splitting.md %}) of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range).
```

Comment:
AIUI they work together:

- the load-based splitting is what cuts up frequently accessed keys into smaller ranges

- while load-based rebalancing is what shuffles the ranges around

Reply:
Oh, in that case I feel that something like "also involves" instead of "operates in conjunction with" is more straightforward and clear here, but I don't feel strongly.